index.html

```diff
@@ -0,0 +1,968 @@
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- section 3 -->
+      <section class="alignment">
+        <div class="title flex block">
+          <div id="div5">
+            <h1>Latest funding rounds on Index.co</h1>
+            <i class="fas fa-angle-right"></i>
+          </div>
+          <div>
+            <a class="red" href="#">Add info |</a
+            ><a class="red" href="#"> Get updates by email |</a
+            ><a class="red" href="#"> See more</a>
+          </div>
+        </div>
+
```

Comment:
#### Mobile Screen Size

- [ ] The layout of these links is different from the original

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91436662-ed980200-e83e-11ea-843d-965a5d249943.png) | ![image](https://user-images.githubusercontent.com/61479648/91436689-fb4d8780-e83e-11ea-86b0-32f5b4823509.png) |